Engine/src/Engine/SceneBuilder/Scene.h

```diff
@@ -99,14 +110,37 @@ namespace Engine
         int m_windowWidth{1920};
         int m_windowHeight{1080};
 
+        //Added this to keep track of the entities created
+        std::unordered_map<std::string, entt::entity> m_entityHandles;
+
+
+        //flag, if true: ui is rendered, if false: no ui is rendered
+        bool showUI = true;
```

Comment:
this should be a build flag, not a local one

Reply:
Right, how do I set it as a build flag?

Reply:
we can leave it as is for now, but add a todo on top so we can change it later. Please also create an associated trello ticket to change this to build flags

Reply:
sounds good